src/Compiler/CompilerUtils.hpp

```diff
@@ -37,47 +37,24 @@ extern llvm::cl::opt<std::string> instrumentONNXOps;
 void setTargetCPU(const std::string &cpu);
 void setTargetTriple(const std::string &triple);
 
-void LoadMLIR(std::string inputFilename, mlir::MLIRContext &context,
-    mlir::OwningModuleRef &module);
-
+std::string compileModuleToObject(
+    const mlir::OwningModuleRef &module, std::string outputBaseName);
 std::string compileModuleToSharedLibrary(
     const mlir::OwningModuleRef &module, std::string outputBaseName);
-
 void compileModuleToJniJar(
     const mlir::OwningModuleRef &module, std::string outputBaseName);
 
 void registerDialects(mlir::MLIRContext &context);
 
 void addONNXToMLIRPasses(mlir::PassManager &pm);
-
 void addONNXToKrnlPasses(mlir::PassManager &pm);
-
 void addKrnlToAffinePasses(mlir::PassManager &pm);
-
 void addKrnlToLLVMPasses(mlir::OpPassManager &pm);
 
 void processInputFile(std::string inputFilename, mlir::MLIRContext &context,
     mlir::OwningModuleRef &module, std::string *errorMessage);
-
 void processInputArray(const void *onnxBuffer, int bufferSize,
     mlir::MLIRContext &context, mlir::OwningModuleRef &module);
 
-onnx_mlir::InputIRLevelType determineInputIRLevel(
```

Comment:
You removed these because you made them static, only used in the file, right?

Reply:
Yes. I made a few function static were they aren't referenced from other compilation units. I think that, in general, is good to limit external functions to just the one that are strictly required.

Reply:
@etiotto @AlexandreEichenberger making these function static means they are not sharable anymore. Other projects using these functions would fail. This is regression.